Fix assumption that pkgconfig lib dir must already exist at configure
time.

--- configure.orig	2021-10-18 02:12:23.000000000 +1100
+++ configure	2021-11-11 21:42:21.000000000 +1100
@@ -4150,6 +4150,7 @@
 
 case x$cf_search_path in
 (x/*)
+    cf_search_path_explicit=yes
 	;;
 (xyes)
 	cf_search_path=
@@ -4235,6 +4236,9 @@
 		break
 	fi
 done
+if test "x$cf_pkg_config_path" = xnone -a "x$cf_search_path_explicit" = xyes; then
+    cf_pkg_config_path=$cf_search_path
+fi
 echo "$as_me:4238: result: $cf_pkg_config_path" >&5
 echo "${ECHO_T}$cf_pkg_config_path" >&6
 
